com.amphi.music.yaml

```diff
@@ -0,0 +1,67 @@
+id: com.amphi.music
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: music
+
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/bin/lib:$LD_LIBRARY_PATH
```

Comment:
this is incorrect.
1. you should need it, your install steps are wrong (see my other note about the prefix)
2. you literally set it to `/app/bin/lib:$LD_LIBRARY_PATH` as environment substitutions are NOT done (this is not a shell).